server/main.js

```diff
@@ -1,67 +1,91 @@
-const express = require('express')
-const path = require('path')
-const webpack = require('webpack')
-const logger = require('../build/lib/logger')
-const webpackConfig = require('../build/webpack.config')
-const project = require('../project.config')
-const compress = require('compression')
+const express = require('express');
+const expressWinston = require('express-winston');
+const winston = require('winston'); // for transports.Console
+const Rotate = require('winston-logrotate').Rotate;
+const path = require('path');
+const webpack = require('webpack');
+const webpackConfig = require('../build/webpack.config');
+const project = require('../project.config');
+const compress = require('compression');
+const logger = require('../build/lib/logger');
 
-const app = express()
-app.use(compress())
+const app = express();
+app.use(compress());
+
+const rotateTransport = new Rotate({
+  file: path.resolve(project.basePath, 'express.log'),
+  colorize: true,
+  timestamp: true,
+  json: false,
+  max: '10m',
+  keep: 5,
+  compress: true
+});
 
 // ------------------------------------
 // Apply Webpack HMR Middleware
 // ------------------------------------
 if (project.env === 'development') {
-  const compiler = webpack(webpackConfig)
+  const compiler = webpack(webpackConfig);	
 
-  logger.info('Enabling webpack development and HMR middleware')
+	logger.info('Enabling webpack development and HMR middleware');
```

Comment:
Seem to be using both logger and winston to log, here. Should we use winston before webpack-hot-middleware, and have it do all the logging in this file?